tests/testthat/test_format_dta.R

```diff
@@ -9,7 +9,18 @@ test_that("Export to Stata", {
 test_that("Import from Stata (read_dta)", {
     expect_true(is.data.frame(import("mtcars.dta", haven = TRUE)))
     # arguments ignored
-    expect_warning(is.data.frame(import("mtcars.dta", haven = TRUE, extraneous.argument = TRUE)))
+    expect_warning(import("mtcars.dta", haven = TRUE, extraneous.argument = TRUE))
+    # tell arg_reconcile() to not show warnings
+    expect_silent(import("mtcars.dta", haven = TRUE,
+                         extraneous.argument = TRUE, .warn = FALSE))
+})
+
+test_that("Import from Stata with extended Haven features (read_dta)", {
+  expect_true(is.data.frame(mtcars_wtam <- import("mtcars.dta", haven = TRUE,
```

Comment:
Note the column names are quoted for the `col_select` argument. Turns out `haven::read_dta()` can accept either quoted or unquoted column names, but `arg_reconcile()` can only handle passing quoted ones at this time. Notice also the concurrent use of another new `read_dta()` argument, `n_max`.